discord-scripts/fix-linear-embed.ts

```diff
@@ -36,7 +36,12 @@ async function createLinearEmbed(
         .setURL(
           `https://linear.app/${teamName}/issue/${issue.identifier}#comment-${commentId}`,
         )
-        .setDescription(comment.body || "No comment body available.")
+        .setDescription(
+          `${(comment.body || "No comment body available.")
+            .split(" ")
+            .slice(0, 50)
+            .join(" ")}...`,
```

Comment:
We should only display the ellipsis if we've actually truncated it. I suggest a helper function that handles this whole behavior:

```typescript
function truncateWords(content: string | undefined, ifBlank: string, maxWords = 50): string {
  if (content === undefined || content.trim() === "") {
    return ifBlank
  }

  const truncatedContent = content.split(" ").slice(0, 50).join(" ")

  if (truncatedContent !== content) {
    return `${truncatedContent}...`
  }

  return content
}
```

Reply:
Just added the `VoiceChannel` support this to this one as well 🫡